### ISSUE TITLE
Steam Link: PreSubmitGpuMs = 0, PostSubmitGpuMs = 0, TotalRenderGpuMs <1

### ISSUE THREAD

**Issue by sergioberg** [ValveSoftware/openvr#1801]
PreSubmitGpuMs = 0, PostSubmitGpuMs = 0, TotalRenderGpuMs <1 in OpenXR games with Steam Link
All the time, no matter what happens in the game. The SteamVR graph shows the same thing.
I've seen this before and usually the problem is on the side of the VR game, but here it's clearly related to the Steam Link. Because it is observed exclusively only when connecting via Steam Link. With other connection methods and when using different wired and wireless headsets, nothing like this is observed.

Game: Bonelab (public branch) (OpenXR)
SteamVR beta 2.1.9
Windows 11
GTX 4090 (546.17)
Quest 3
Steam Link 2.0.1

**Comment by alexeiw123** [comment]
> Replying to https://github.com/ValveSoftware/openvr/issues/1801#issuecomment-3085261601

I can't replicate this unfortunately

PSVR2 in iRacing.

**Comment by sergioberg** [comment]
> Replying to https://github.com/ValveSoftware/openvr/issues/1801#issuecomment-3102692354

PSVR2 is not using Steam Link. It's for Quest 2/3
and iRacing is not OpenXR game

**Comment by alexeiw123** [comment]
> Replying to https://github.com/ValveSoftware/openvr/issues/1801#issuecomment-3102692354

AH, OK - I got to this issue from your post at https://steamcommunity.com/app/908520/discussions/0/4139438760460112743/#c595154681972519866

iRacing uses OpenXR, but if this is specific to steamlink then unrelated.

**Comment by sergioberg** [comment]
> Replying to https://github.com/ValveSoftware/openvr/issues/1801#issuecomment-3105063374

Is this your topic here: https://steamcommunity.com/app/908520/discussions/0/546746241084665913/#c596286923596891222
?
If NOT try check that OpenXR layers are you using. Especially Racelab.

**Comment by alexeiw123** [comment]
> Replying to https://github.com/ValveSoftware/openvr/issues/1801#issuecomment-3337429935

Yep that was me, but I think closing out the loop here may help someone else trying to solve the same issue.
